build.gradle.kts

```diff
@@ -29,152 +16,164 @@ repositories {
     maven { url = uri("https://repo.gradle.org/gradle/libs-releases/") }
 }
 
-fun SourceDirectorySet.configureTestSources() {
-    srcDir("src/jvm/test")
-    srcDir("src/jvm/test-trace-debugger-integration")
-
-    val jdkToolchainVersion: String by project
-    if (jdkToolchainVersion.toInt() >= 11) {
-        srcDir("src/jvm/test-jdk11")
-    } else {
-        srcDir("src/jvm/test-jdk8")
-    }
-}
-
 kotlin {
-    @OptIn(ExperimentalKotlinGradlePluginApi::class)
     compilerOptions {
         allWarningsAsErrors = true
     }
+}
 
-    jvm {
-        withJava()
-
-        val main by compilations.getting
-        val test by compilations.getting
-        val integrationTest by compilations.creating {
-            defaultSourceSet {
-                associateWith(main)
-                associateWith(test)
-            }
-        }
+sourceSets {
+    main {
+        java.srcDirs("src/jvm/main")
```

Comment:
Should we change the folder to `src/main` and change other folders correspondingly?

Reply:
Note: we can do that at some point later, when we do not have a lot of PRs in Lincheck